core/src/Revolution/Sources/modMediaSource.php

```diff
@@ -2238,6 +2238,10 @@ protected function getImageDimensions($path, $ext)
                 } else {
                     return false;
                 }
+                if($height > 80){
```

Comment:
It's better to use `$this->ctx->getOption('filemanager_thumb_height', 80)` instead of a hardcoded value of 80.

Reply:
should I also add **filemanager_thumb_height**  system setting by default or not?